wordssite/words/migrations/0002_choice_word_word_translation_delete_translation_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.0.4 on 2022-05-03 14:49
```

Comment:
Once you're done with the models.py file - delete all files in migrations folder and run `python manage.py makemigrations` again - it will create one file with all the changes.